index.html

```diff
@@ -10,7 +10,68 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
+
+<section>
```

Comment:
[nit] Try to indent consistently. This whole `<section>` should be tabbed one to the right.